pymaker/dss.py

```diff
@@ -384,6 +384,57 @@ def line(self) -> Rad:
         """ Total debt ceiling """
         return Rad(self._contract.functions.Line().call())
 
+    def flux(self, ilk: Ilk, src: Address, dst: Address, wad: Wad) -> Transact:
+        """Move Ilk balance in Vat from source address to destiny address
+
+        Args:
+            ilk: Identifies the type of collateral.
+            src: Source of the collateral (address of the source).
+            dst: Destiny of the collateral (address of the recipient).
+            wad: Amount of collateral to move.
+        """
+        assert isinstance(ilk, Ilk)
+        assert isinstance(src, Address)
+        assert isinstance(dst, Address)
+        assert isinstance(wad, Wad)
+
+        flux_args = [ilk.toBytes(), src.address, dst.address, wad.value]
+        return Transact(self, self.web3, self.abi, self.address, self._contract, 'flux', flux_args)
+
+    def move(self, src: Address, dst: Address, wad: Wad) -> Transact:
```

Comment:
The [contract](https://github.com/makerdao/dss/blob/master/src/vat.sol#L151) seems to take a `Rad`.

Reply:
This is already fixed